CHANGELOG.md

```diff
@@ -5,6 +5,70 @@
 <!-- For unreleased changes, see entries in .chloggen -->
 <!-- next version -->
 
+## v0.136.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.136.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.136.0)
+and the [opentelemetry-collector-contrib v0.136.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.136.0) releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+- (Contrib) `countconnector`: Upgrade profiles proto to 1.8.0 ([#42526](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/42526))
+- (Contrib) `internal/kafka`: Disable Kafka client metrics ([#42662](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/42662))
+  We disable publishing Kafka client metrics to brokers, since they were not
+  added intentionally and may lead to log spam when the broker does not really
+  support metrics. We may make this configurable in the future.
+- (Contrib) `pkg/ottl`: Upgrade profiles proto to 1.8.0 ([#42526](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/42526))
+  See proto changelog. https://github.com/open-telemetry/opentelemetry-proto/blob/main/CHANGELOG.md#180---2025-09-02
+- (Contrib) `transformprocessor`: Upgrade profiles proto to 1.8.0 ([#42526](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/42526))
```

Comment:
```suggestion
```
Not sure if otlp profiling matters to our customers at present.

Reply:
Lately we've just been leaving everything in just in case. It's fine with me to leave it.